src/test/java/uk/gov/justice/hmpps/prison/service/BookingServiceTest.java

```diff
@@ -367,16 +367,37 @@ public void getBookingIEPSummary_offenderHasNoIepSummary() {
     public void getBookingIEPSummary_multipleBooking_globalSearchUser() {
         when(authenticationFacade.isOverrideRole(anyString(), anyString(), anyString(), anyString(), anyString())).thenReturn(true);
         when(bookingRepository.getBookingIEPDetailsByBookingIds(anyList())).thenReturn(Map.of(-5L, List.of(PrivilegeDetail.builder().iepDate(LocalDate.now()).build())));
+        when(offenderBookingRepository.findById(anyLong())).thenReturn(Optional.of(OffenderBooking.builder().location(AgencyLocation.builder().id("MDI").build()).build()));
```

Comment:
Had to update existing tests because I introduced `offenderBookingRepository.findById` and without this mock they were throwing `EntityNotFoundException`

Reply:
Does this definitely only get called if no IEP found?

Reply:
Ignore I can see that it does